client/test_commands/test_altbn128_mixer_base.py

```diff
@@ -0,0 +1,148 @@
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000001",
+        2),
+    int(
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000010",
+        2),
+]
+
+HSIG = int(
+    "1010000000000000000000000000000000000000000000000000000000000000"
+    "0000000000000000000000000000000000000000000000000000000000000000"
+    "0000000000000000000000000000000000000000000000000000000000000000"
+    "0000000000000000000000000000000000000000000000000000000000101111",
+    2)
+
+HTAGS = [
+    int(
+        "1100000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000110010",
+        2),
+    int(
+        "1110000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000111011",
+        2),
+]
+
+VPUB = (0x5555555555555500, 0x00eeeeeeeeeeeeee)
+
+# 255                                         128         64           0
+# |<empty>|<h_sig>|<nullifiers>|<msg_auth_tags>|<v_pub_in>)|<v_pub_out>|
+RESIDUAL_BITS = int(
+    "101"  # h_sig
+    "010"  # nf_1
+    "001"  # nf_0
+    "111"  # htag_1
+    "110"  # htag_0
+    "0101010101010101010101010101010101010101010101010101010100000000"  # vin
+    "0000000011101110111011101110111011101110111011101110111011101110",  # vout
+    2)
+
+PACKED_PRIMARY_INPUTS = \
+    [ROOT] + COMMITMENTS + NULLIFIERS + [HSIG] + HTAGS + [RESIDUAL_BITS]
+
+
+def test_assemble_nullifiers(mixer_instance: Any) -> None:
+    # Test retrieving nullifiers
+    print("--- testing ", "test_assemble_nullifiers")
+    for i in range(JS_INPUTS):
+        res = mixer_instance.functions.\
+            assemble_nullifier_test(i, PACKED_PRIMARY_INPUTS).call()
+        val = int.from_bytes(res, byteorder="big")
+        assert val == NULLIFIERS[i], f"expected: {NULLIFIERS[i]}, got: {val}"
+
+
+def test_assemble_hsig(mixer_instance: Any) -> None:
+    # Test retrieving commitments
+    print("--- testing ", "test_assemble_hsig")
+    res = mixer_instance.functions.\
+        assemble_hsig_test(PACKED_PRIMARY_INPUTS).call()
+    hsig = int.from_bytes(res, byteorder="big")
+    assert hsig == HSIG, f"expected: {HSIG}, got {hsig}"
+
+
+def test_assemble_vpub(mixer_instance: Any) -> None:
+    # Test retrieving commitments
+    print("--- testing ", "test_assemble_vpub")
```

Comment:
`test_..._vpub` -> `assemble_public_values_test`

Reply:
(as above.  `test_assemble_vpub` seems more suitable for printing.)